platform/centec-arm64/sonic-platform-modules-e550/24t16y/modules/dal_kernel.c

```diff
@@ -0,0 +1,1825 @@
+/**
+ @file dal_kernal.c
```

Comment:
what is dal_kernel, is this you asic driver? if yes, why do you need the same code for different platform?

Reply:
> what is dal_kernel, is this you asic driver? if yes, why do you need the same code for different platform?

yes. I will modify this.